.github/workflows/publish-packages.yml

```diff
@@ -0,0 +1,67 @@
+name: Publish packages
+
+on:
+  workflow_call:
+    inputs:
+      upload_to_test:
+        required: true
+        type: boolean
+    secrets:
+      TEST_PYPI_API_TOKEN:
+        required: false
+      PYPI_API_TOKEN:
+        required: false
+
+jobs:
+  publish-package:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out repo
+      uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        # We must explicitly install the requirements so that we can force
+        # installation of the local wheel below (in case the version conflicts
+        # with published wheels, typically only possible during testing).
+        # We could remove this when finalizing the workflow and just rely on
```

Comment:
Let’s remove the second half of this comment that suggests an alternative. I like the strategy as written.

Reply:
Sounds good, figured I'd put in the comments and see what reviewers thought.